andela/activo-web
# #159568519 Edit a role

**What does this PR do?**

This PR ensures that a user should be able to edit any role on Activo

**Description of Task to be completed?**

- Modify `userRolesPage` to accommodate editing a role.
- Create action to fetch the details of the the role selected
- Create action to update the details of the the selected role.

**How should this be manually tested?**

- Visit settings tab
- Click on the `Edit` button on any role.
- An `EDIT ROLE` modal should pop up with the information on the selected role prepopulated
- Edit the role and save changes.

**Any background context you want to provide?**

xxx

**What are the relevant pivotal tracker stories?**

[#159568519](https://www.pivotaltracker.com/n/projects/2170023/stories/159568519)

**Screenshots (if appropriate)**

<img width="1440" alt="screen shot 2018-08-18 at 3 45 36 pm" src="https://user-images.githubusercontent.com/28443813/44300363-cdebac00-a2fd-11e8-98bf-2bd1878cb9be.png">


**Questions:**

xxx
